### DISCUSSION TITLE
"light" mode being overridden by "dark" class

### DISCUSSION THREAD

**Discussion by kedarjoyner** [jpanther/congo#466] [Q&A] [answered]
Hey there! I downloaded the theme yesterday using the recommended module method. I'm unable to get "light" mode to work for some reason. 

My `params.toml` file looks like this...
<img width="983" alt="Screenshot 2023-02-05 at 10 32 19 AM" src="https://user-images.githubusercontent.com/12139428/216831879-200cf8ed-18d9-4fa6-b43d-1510c7aadca8.png">

I can see that `light` is being set in the HTML, but the class `dark` is overriding this on the same element:
<img width="809" alt="Screenshot 2023-02-05 at 10 30 40 AM" src="https://user-images.githubusercontent.com/12139428/216831906-e16bef2c-f91c-44fe-bdbd-fac459e7bcf9.png">

Any ideas what I might be doing wrong? Thanks, loving the theme so far! <3

**Comment by kedarjoyner** [comment, marked as answer]
Ah! I think I figured this out. I had the switcher on and it was saving/cacheing my settings, I think. Once I toggled it to light, and then turned the switcher off, it started enforcing these settings. 

🎉

**Reply by jpanther** [reply]
The switcher uses local storage to save the preferred appearance state so you either have to clear out the local storage, or you can also right click on the switcher icon which will delete the saved preference.

**Reply by kedarjoyner** [reply]
Thank you! It's helpful to hear the technical explanation. I think this behavior is intuitive for more experienced devs (I wish that was me lol), so I'll do a pull request to add to this documentation for us baby devs. I'm learning so, so, so much just using this theme 👩‍💻

**Reply by jpanther** [reply]
It's not that it's about intuitive behaviour, it's more that you're mixing the reality of a development environment with a production deployment. Normal users should not encounter an issue such as this as you would not be turning the appearance switcher on and off all the time which means they aren't going to get settings persisted in their local storage.

**Reply by kedarjoyner** [reply]
Ahhh, got it! That makes a lot more sense. I was definitely doing a lot of on/off switching in my local environment. Thanks for the thorough explanation!
